iothub/device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/mqtt/MqttDirectMethod.java

```diff
@@ -27,14 +27,14 @@ class MqttDirectMethod extends Mqtt
     private static final String BACKSLASH = "/";
     private static final String QUESTION = "?";
 
-    private static final String METHOD = "$iothub/methods/";
-    private static final String POST = METHOD + "POST";
-    private static final String RES = METHOD + "res";
-    private static final String REQ_ID = QUESTION + "$rid=";
+    private static final String METHOD = "$iothub/methods";
```

Comment:
Same comment about the number of constants as I made in #1713. We don't need a constant for every character/word here.